src/main/java/expiryeliminator/data/IngredientList.java

```diff
@@ -77,6 +79,43 @@ public Ingredient findIngredient(String ingredientDescription) {
         return ingredients.get(ingredientDescription);
     }
 
+    /**
```

Comment:
This is a good implementation to check if the ingredient is expiring soon. This feature is also one of the most critical parts of the app.